README.md

```diff
@@ -30,9 +30,8 @@ If you have questions or need help, please check out our documentation for a [li
 
 ## conda-forge dev meetings
 
-Our documentation contains a section with [minutes from previous dev meetings]([https://conda-forge.org/docs/minutes/00_intro.html]). These meetings occur every two weeks on Wednesday from 17:00-18:00 UTC.
-A link to the google calendar item can be found [here](https://calendar.google.com/event?action=TEMPLATE&tmeid=Z2lraDk2a205cGUxdDkxYmNybXQxMGIxMGtfMjAxOTA3MjRUMTcwMDAwWiBzY29wYXR6QG0&tmsrc=scopatz%40gmail.com&scp=ALL).
+We hold biweekly meetings every second Wednesday from 17:00-18:00 (UTC). Feel free to stop by then with the help of the [link](https://calendar.google.com/calendar/u/0/r/eventedit/copy/Z2lraDk2a205cGUxdDkxYmNybXQxMGIxMGtfMjAxOTA3MjRUMTcwMDAwWiBzY29wYXR6QG0/c2hsb2thcHJpbmNlc3MxMDFAZ21haWwuY29t?scp=ALL&sf=true).
```

Comment:
I'd remove "then" from after "stop by".

And also replace "the" with "this" before "link". It would even be better to do "this [calendar link]" instead, to be absolutely clear.